TagsCloudContainer/Infrastructure/Tag.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace TagsCloudContainer.Infrastructure;
+
+public record Tag

```

Comment:
```suggestion
public record Tag(double Frequency, string Text);
```

Вроде это эквивалент)

Reply:
Видимо по аналогии с наследниками поправлял, в них пришлось делать конструкторы, чтобы унаследоваться